### PR TITLE
[Dashboard] Remove the view data

### PR DESCRIPTION
**Pull request by rkooo567**
Signed-off-by: SangBin Cho <rkooo567@gmail.com>

## Why are these changes needed?

View data is unnecessary in the new dashboard. Same for node detail view.

When the node is idle at 250 nodes, CPU usage is about 20~30%, and I found most of usage is from MessageToDict. Since we have lots of view data, I assume most of overhead is from view data.

203-2022-10-29T10_38_19-07_00

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/modules/node/tests/test_node.py [hunk 1]
```diff
@@ -180,59 +178,6 @@ def check_mem_table():
...
-        def get_obj(self):
-            return ray.get(self.obj_ref)
-
-    actors = [ActorWithObjs.remote() for _ in range(2)]  # noqa
-    timeout_seconds = 20
-    start_time = time.time()
-    last_ex = None
-
-    def check_node_details():
-        resp = requests.get(f"{webui_url}/nodes?view=details")
```
Comment (rkooo567) [inline dashboard/modules/node/tests/test_node.py:203]: this endpoint is just for the old dashboard

**Review by rickyyx — COMMENTED** [review]
Do we have a flame graph after this PR is applied?

**Comment by rkooo567** [comment]
> Do we have a flame graph after this PR is applied?

I can run it after we merge the PR. This data is unnecessary anyway and we should remove it

**Review by rickyyx — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/jjjjjohn/">jjjjjohn</a> on <a href="https://giphy.com/gifs/skeleton-sweep-swiffer-3o7TKqTOqevMiJAwU0">GIPHY</a></div>

**Review by ericl — APPROVED** [review]

**Comment by rkooo567** [comment]
@ericl @rickyyx The idle cpu usage has drastically reduced after this PR, and the flamegraph (no more message to dict overhead!)

204-2022-11-02T19_32_10-07_00

**Comment by alanwguo** [comment]
amazing!

**Comment by ericl** [comment]
Seems the remaining flame graph is doing a ton of extra work in internal_kv_get/put for update_nodes()--- at a glance this seems to be unnecessary repeated reads of the agent port / storing the head node id. We could probably eliminate most of that with a couple if statement rearrangements.

**Pull request merged**